js/lib/alert-bar.js

```diff
@@ -0,0 +1,16 @@
+var $ = require('jquery')
+var callData = require('./communityCall.json')
+
+function callTime () {
+  var now = new Date().getHours()
+  var startTime = parseInt(callData.time, 10)
```

Comment:
This doesn't take into account the time zone so the banner will appear for the weekly call 2 hours before 5pm in whatever time zone you're in!

For example, (if I do the date maths correctly) in PDT the call is at 10am, but the banner will show between 3pm - 5pm PDT.

This is because `new Date()` will give you back a:

> `Date` object for the current date and time according to system settings for timezone offset
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#Description

You could use [`getUTCHours`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getUTCHours) and [`getUTCDay`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getUTCDay) instead?